x-pack/plugins/ml/public/application/jobs/components/custom_url_editor/utils.js

```diff
@@ -109,6 +115,7 @@ export function isValidCustomUrlSettings(settings, savedCustomUrls) {
 }
 
 export function buildCustomUrlFromSettings(settings) {
+  console.log('settings', settings);
```

Comment:
Do we want to keep this console log in?

Reply:
Thanks for catching that! Removed here https://github.com/elastic/kibana/pull/141871/commits/d20a4b024e48324e4d02a45381f9db07e042d916